### DISCUSSION TITLE
Generation of Summarized Experiment following normalization step

### DISCUSSION THREAD

**Discussion by sylestiel** [stuart-lab/signac#1283] [Q&A] [answered]
Can you provide some guidance on converting from Chromatin Assay to Summarized Experiment or Ranged Summarized Experiment. 
scATAC-seq peak counts as a SummarizedExperiment object, where the rowRanges are fixed-width accessibility peak ranges (typically 300 bp or so), and the assay object is a sparseMatrix of Tn5-instertion counts per peak per cell. 

Thanks!

**Comment by timoast** [comment, marked as answer]
You can convert to a SingleCellExperiment object (which is based on a RangedSummarizedExperiment) using the `as.SingleCellExperiment()` function as shown here: https://satijalab.org/seurat/articles/conversion_vignette.html#converting-to-from-singlecellexperiment

**Reply by MartijnLvandenBroek** [reply]
I tried to do the same, and while it worked to convert the Chromatin Assay to a SingleCellExperiment object, I lost the GRanges information which I also need in my SingleCellExperiment. Is there currently a solution to keep the GRanges information in the SingleCellExperiment?
